index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Project</title>
+</head>
+<body>
+<header>
+ <nav class="navbar">
+   <div class="logo">
+     <a href="#">My logo</a>
+   </div>
+   <div class="icon"><img src="/images/Union.png" alt="menu icon"></div>
+</nav>

```

Comment:
Optional suggestions but I highly recommend you to work on it before you merge

- [ ] Please hamburger icon as required instead of menu icon text